### PR TITLE
Made a script that when you run it will install APX 2.0

### PR DESCRIPTION
**Pull request by GabsEdits**
From blog, https://github.com/Vanilla-OS/website/pull/104

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,13 @@ make install DESTDIR=$HOME/altroot
 make install-manpages DESTDIR=$HOME/altroot
 ```
 
+
+### Installing using or script
```
Comment (mirkobrombin) [inline README.md:132]: Possible typo?

File: README.md [hunk 2]
```diff
@@ -128,6 +128,13 @@ make install DESTDIR=$HOME/altroot
 make install-manpages DESTDIR=$HOME/altroot
 ```
 
+
+### Installing Apx using an shell script
+This script installs Apx automatically when you run the script, just make sure you have  `curl` and `git` installed, once you do clone this project and run the shell script
+```bash
```
Comment (kbdharun) [inline README.md:134]:
```suggestion
### Installing Apx using a shell script

This script installs and sets up Apx automatically when you execute it, make sure you have  `curl` and `git` installed, then clone this project and run it.

```bash
```

Made a few minor fixes to the wording and also fixed Markdown spacing.

File: install.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+echo "Please enter or create a working directory:"
```
Comment (kbdharun) [inline install.sh:3]:
```suggestion
echo "Please enter or create a working directory."
```

File: install.sh [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+read -r WORK_DIR
+
+mkdir -p "$WORK_DIR" # Creates the working directory if non existing
+
+cd "$WORK_DIR"
+
+curl -LO https://github.com/89luca89/distrobox/archive/refs/tags/1.5.0.2.tar.gz
+tar -xzf 1.5.0.2.tar.gz
```
Comment (kbdharun) [inline install.sh:11]:
```suggestion
curl -LO https://github.com/89luca89/distrobox/archive/refs/tags/1.7.2.1.tar.gz
tar -xzf 1.7.2.1.tar.gz
```

File: install.sh [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+curl -LO https://github.com/Vanilla-OS/apx/releases/download/continuous/apx.tar.gz
+tar -xzf apx.tar.gz
+mv apx "$HOME/.local/bin/apx2"
+chmod +x "$HOME/.local/bin/apx2"
+
+mkdir -p "$HOME/.config/apx"
+echo '{
+  "apxPath": "'"$HOME/.local/share/apx/"'",
+  "distroboxpath": "'"$WORK_DIR/distrobox-1.5.0.2/distrobox"'",
```
Comment (kbdharun) [inline install.sh:21]:
```suggestion
  "distroboxpath": "'"$WORK_DIR/distrobox-1.7.2.1/distrobox"'",
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for these files.

**Comment by axtloss** [comment]
We provide a makefile in apx, it would be better to just make the script use that instead.

**Comment by GabsEdits** [comment]
> We provide a makefile in apx, it would be better to just make the script use that instead.

I guess it would be better. That would also require me to update the old blog with the new script. This Pull request currently uses the old script that was made when the blog was released, with a few modifications to make the script more user friendly.

**Comment by kbdharun** [comment]
The v2 branch was migrated to the main branch a while ago, If required then this script along with the above suggestions should be proposed for the default branch (main).

**Pull request closed without merging**